Formula/phive.rb

```diff
@@ -0,0 +1,19 @@
+class Phive < Formula
+  desc "Phar Installation and Verification Environment (PHIVE)"
+  homepage "https://phar.io"
+  url "https://github.com/phar-io/phive/releases/download/0.14.4/phive-0.14.4.phar"
+  sha256 "b6f95e2791a17531f6ba652ce6e4288f35fb73883c472af1d3758f148fc4ed3a"
+  license "BSD-3-Clause"
+
+  bottle :unneeded
+
+  depends_on "php@7.2" => :test
```

Comment:
`phive` requires PHP 7.2 - am I doing this right here?

Reply:
`phive` requires PHP 7.2 ___or later___.

Reply:
```suggestion
  depends_on "php"
```

I had to do the same on #63294.

Reply:
Thank you, @derrabus!